presto/dataops/utils.py

```diff
@@ -60,8 +61,14 @@ def construct_single_presto_input(
     if dynamic_world is None:
         dynamic_world = torch.ones(num_timesteps) * (DynamicWorld2020_2021.class_amount)
 
+    keep_indices = [idx for idx, val in enumerate(BANDS) if val != "B9"]
+    mask = mask[:, keep_indices]
+
     if normalize:
-        keep_indices = [idx for idx, val in enumerate(BANDS) if val != "B9"]
-        mask = mask[:, keep_indices]
         x = S1_S2_ERA5_SRTM.normalize(x)
```

Comment:
Can you add a comment here that `.normalize` includes `x = x[:, keep_indices]`? :)

Reply:
added!